app/styles/partials/rails.scss

```diff
@@ -0,0 +1,57 @@
+@media screen and (min-width : 1200px) {
```

Comment:
Keep a max-width of `1600px` here. because, after `1600px` the normal width itself is alright